testsupport/space.go

```diff
@@ -100,12 +112,6 @@ func CreateSpaceWithBinding(t *testing.T, awaitilities wait.Awaitilities, mur *t
 	return space, spaceBinding
 }
 
-// CreateAndVerifySpace does the same as CreateSpace plus it also verifies the provisioned resources in the cluster using function VerifyResourcesProvisionedForSpace
-func CreateAndVerifySpace(t *testing.T, awaitilities wait.Awaitilities, opts ...SpaceOption) *toolchainv1alpha1.Space {
-	space, _, _ := CreateSpace(t, awaitilities, opts...)
-	return VerifyResourcesProvisionedForSpace(t, awaitilities, space.Name)
-}
```

Comment:
The same question as before - what is the reason for removing this function?

Reply:
trying to reduce the amount of utility funcs, because it's getting harder and harder to follow up, IMO

Reply:
We need to improve that for sure - some fancy fluent-API would be really nice. 
Those utility functions were introduced to eliminate the repetitive lines that are called together over and over again.
But if removing this single utility function helps with the code readability, then I'm fine with that.